frontend/src/scenes/onboarding/OnboardingProductConfiguration.tsx

```diff
@@ -14,53 +42,88 @@ export const OnboardingProductConfiguration = ({
-            <h2 className="pt-2">Options</h2>
-            {configOptions?.map((option: ProductConfigOption, idx) => (
-                <React.Fragment key={idx}>
-                    <LemonDivider className="my-4" />
-                    <div className="grid gap-4 grid-cols-[minmax(min-content,_2fr)_minmax(140px,_1fr)_minmax(min-content,_3fr)] items-center">
-                        <label className="text-base font-semibold">{option.title}</label>
-                        {option.type == 'toggle' ? (
-                            <LemonSwitch
-                                data-attr="opt-in-session-recording-switch"
-                                onChange={(checked) => {
-                                    setConfigOptions(
-                                        configOptions.map((o) =>
-                                            o.teamProperty === option.teamProperty ? { ...o, value: checked } : o
-                                        )
-                                    )
-                                }}
-                                className="self-center"
-                                fullWidth={true}
-                                checked={option.value || false}
-                            />
-                        ) : (
-                            <div className="flex justify-between items-center mb-1 gap-x-4">
-                                <LemonSelect
-                                    dropdownMatchSelectWidth={false}
-                                    onChange={(v) => {
-                                        setConfigOptions(
-                                            configOptions.map((o) =>
-                                                o.teamProperty === option.teamProperty ? { ...o, value: v } : o
-                                            )
-                                        )
-                                    }}
-                                    options={option.selectOptions || []}
-                                    value={option.value}
-                                />
+            <div className="mt-6">
+                <h2 className="pt-2">Options</h2>
+                {combinedList.map((item, idx) => (
+                    <React.Fragment key={idx}>
+                        <LemonDivider className="my-4" />
+                        <div className="grid gap-4 grid-cols-3">
+                            <div className="col-span-2">
+                                <label className="text-base font-semibold">{item.title}</label>
+                                <p className="prompt-text mt-2 mb-0 ">{item.description}</p>
+                            </div>
+                            <div className="flex justify-end">
+                                {item.type === 'toggle' ? (
+                                    <LemonSwitch
+                                        data-attr="opt-in-session-recording-switch"
```

Comment:
prob don't want this as the data-attr (I see it was existing, but worth changing to something that makes sense)